src/site/content/en/accessible/color-and-contrast/index.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: Color and contrast
+authors:
+  - dgash
+  - megginkearney
+  - rachelandrew
+  - robdodson
+date: 2020-05-28
+description: |
+  If you have good vision, it's easy to assume that everyone perceives colors, or text legibility, the same way you do — but of course that's not the case.
+tags:
+  - blog
+  - accessibility
+---
+
+If you have good vision, it's easy to assume that everyone perceives colors,
+or text legibility, the same way you do—but of course that's not the case.
+
+As you might imagine,
+some color combinations that are easy for some people to read are difficult or impossible for others.
+This usually comes down to color contrast,
+the relationship between the foreground and background colors' luminance.
+When the colors are similar, the contrast ratio is low;
+when they are different, the contrast ratio is high.
+
+The [WebAIM guidelines](https://webaim.org/standards/wcag/) recommend an AA (minimum) contrast ratio of 4.5:1 for all text.
+An exception is made for very large text (120-150% larger than the default body text),
+for which the ratio can go down to 3:1. Notice the difference in the contrast ratios shown below.
+
+<figure class="w-figure">
+  <img src="./contrast-ratios.jpg" alt="An image showing the different contrast ratios">
+</figure>
+
+The contrast ratio of 4.5:1 was chosen for level AA
+because it compensates for the loss in contrast sensitivity
+usually experienced by users with vision loss equivalent to approximately 20/40 vision.
+20/40 is commonly reported as typical visual acuity of people at about age 80.
+For users with low vision impairments or color deficiencies,
+we can increase the contrast up to 7:1 for body text.
+
+You can use the Accessibility Audit in Lighthouse to check your color contrast.
+Open DevTools, click Audits, and select Accessibility to run the report.
+
+<figure class="w-figure">
+  <img src="./accessibility-audit.jpg" alt="An image showing the different contrast ratios">
+</figure>
+
+For a more complete report, install the [Accessibility Insights Extension](https://accessibilityinsights.io/).
+One of the checks in the Fastpass report, is color contrast.
+You will get a detailed report of any failing elements.
+
+<figure class="w-figure">
+  <img src="./fastpass-contrast.jpg" alt="The report in Accessibility Insights">
```

Comment:
Might want to update to latest lighthouse report. Also the screenshot is a bit fuzzy so doing a new version may also make it easier to read for folks.

Reply:
Is this supposed to be a screenshot of Accessibility Insights? It looks like Lighthouse 

![image](https://user-images.githubusercontent.com/4713486/83207135-4dc7bc80-a107-11ea-8007-f7cc02b67a9f.png)